controllers/users.js

```diff
@@ -63,8 +64,15 @@ class Users {
    */
   static async activateUserAccount(req, res) {
     const { token } = req.params;
-    const { email } = jwt.verify(token, process.env.SECRET);
+    const { id, email } = jwt.verify(token, process.env.SECRET);
     await User.update({ activated: 1 }, { where: { email } });
+
+    // create subscribers row
+    await subscribers.create({
```

Comment:
@fridolinho I think making every user who updated his account a subscriber and adds him/her to the subscribers' table automatically would be like a duplication, It would be better if you add a user as a subscriber after like when he favorites or comment and is going to receive notifications later.

Reply:
You have get it wrong, here i am creating a row with 0 subscribers, and it will be updated when a user is followed, but am changing it to be created when a user is followed for the first time.